src/installscreen.cpp

```diff
@@ -56,7 +56,7 @@ InstallScreen::InstallScreen(PackageBackend *backend, QString *tmpFileName, QObj
     installLayout->setSpacing( 20 );
 
     foreach( QUrl iter, m_backend->repositories() ) {
-        QLabel *sourceLabel = new QLabel( QString( "Added Source: %1" ).arg( iter.toString() ) );
+        QLabel *sourceLabel = new QLabel( i18n("Added Source: %1", iter.toString()) );
```

Comment:
I would prefer to use the more standard i18n("Added Source: %1").arg( iter.toString() )  form than passing the argument to the i18n call. The way you did it is ok, but I think it's not the usual way to do it. Do you have a strong opinion about using that form?